subsys/net/ip/icmpv4.c

```diff
@@ -208,6 +208,7 @@ int net_icmpv4_send_echo_request(struct net_if *iface,
 
 	echo_req->identifier = htons(identifier);
 	echo_req->sequence   = htons(sequence);
+	echo_req->time_stamp = htonl(k_cycle_get_32());
```

Comment:
so line 191 you would need to add sizeof(u32_t) in addition of the echo_req

then here, remove that line. And after line 213 below just: net_pkt_write_be32_new(k_cycle_get_32());